<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Buscador</class>
 <widget class="QWidget" name="Buscador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscador de peliculas</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QPushButton" name="info">
     <property name="text">
      <string>Descriá¹•cion</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QListWidget" name="lista_resutado"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_pelicula">
     <property name="text">
      <string>Ingrese el nombre de la pelicula:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="boton_buscar_pelicula">
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="input_pelicula"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="boton_actor">
     <property name="text">
      <string>Buscar por actor</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_resultado">
     <property name="text">
      <string>Resultado:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_buscar_pelicula</sender>
   <signal>clicked()</signal>
   <receiver>Buscador</receiver>
   <slot>_BuscadorPeliculas__buscar_pelicula()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_actor</sender>
   <signal>clicked()</signal>
   <receiver>Buscador</receiver>
   <slot>_BuscadorPeliculas__abri_busqueda_por_actor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>info</sender>
   <signal>clicked()</signal>
   <receiver>Buscador</receiver>
   <slot>_BuscadorPeliculas__mostrar_info_pelicula()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_BuscadorPeliculas__buscar_pelicula()</slot>
  <slot>_BuscadorPeliculas__abri_busqueda_por_actor()</slot>
  <slot>_BuscadorPeliculas__mostrar_info_pelicula()</slot>
 </slots>
</ui>
